packages/wallet-stamper/src/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const SIGNATURE_SCHEME_TK_API_SECP256K1 =
+  "SIGNATURE_SCHEME_TK_API_SECP256K1";
+export const SIGNATURE_SCHEME_TK_API_ED25519 =
+  "SIGNATURE_SCHEME_TK_API_ED25519";
+export const WALLET_TYPE_EVM = "evm";
+export const WALLET_TYPE_SOLANA = "solana";
+export const STAMP_HEADER_NAME = "X-Stamp";
```

Comment:
Doesn't have to be a now thing, but could be useful to have a separate header type here, such that we have the following:

```
X-Stamp: API keys
X-Stamp-Webauthn: webauthn/yubikey
X-Stamp-Wallet: crypto wallets
```

Reply:
ooo yes I like that idea cc @robin-turnkey